app/scripts/lib/assertion.js

```diff
@@ -5,11 +5,12 @@
 define(function (require, exports, module) {
   'use strict';
 
+  var Duration = require('durationJs');
   var jwcrypto = require('vendor/jwcrypto');
   var P = require('lib/promise');
 
-  var CERT_DURATION_MS = 1000 * 60 * 60 * 6; // 6hrs
-  var ASSERTION_DURATION_MS = 1000 * 3600 * 24 * 365 * 25; // 25 years
+  var CERT_DURATION_MS =  new Duration('6h').milliseconds();
+  var ASSERTION_DURATION_MS = new Duration('52w').milliseconds() * 25; //25 years
```

Comment:
@pdehaan How are you feeling about that 25 day difference? Should we keep the numbers as they are or should we make a lib that calculates the years the same way as moment?